src/Adapter/Shop/Context.php

```diff
@@ -159,13 +159,14 @@ public function setShopGroupContext($id)
     }
 
     /**
-     * Update Multishop context for only one shop group.
-     *
-     * @param int $id Shop id to set in the current context
+     * Update Multishop context to global context
      */
-    public function setAllContext($id)
+    public function setAllContext($id = null)
     {
-        Shop::setContext(Shop::CONTEXT_ALL, $id);
+        if (null !== $id) {
+            trigger_error('id param is deprecated since 8.x', E_USER_DEPRECATED);
```

Comment:
```suggestion
            @trigger_error(
                'The parameter $id is deprecated since version 8.0.0 and will be removed in the next major version.',
                E_USER_DEPRECATED
            );
```

Reply:
@Progi1984 what is the next major version ? 😀

Reply:
The 9.0.0.